.env.base

```diff
@@ -175,5 +175,5 @@ REACT_APP_ZERION_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zerion
 REACT_APP_ZRX_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zrx/
 
 # chainflip
-REACT_APP_CHAINFLIP_API_KEY=6ba154d4-e219-472a-9674-5fa5b1300ccf
+REACT_APP_CHAINFLIP_API_KEY=09bc0796ff40435482c0a54fa6ae2784
```

Comment:
Where does this API key comes from? I know you've been helping to set it up but is the DAO controlling it?

Reply:
Will ping you on discord to protect the safety of the controller